Resources/views/sckenhancedselection_content_field.html.twig

```diff
@@ -1,7 +1,16 @@
-{% block ngclasslist_field %}
+{% block sckenhancedselection_field %}
+{% set available_options = fieldSettings.options %}
 {% spaceless %}
-    {% for identifier in field.value.identifiers %}
-        {{ identifier }}{% if not loop.last %}, {% endif %}
-    {% endfor %}
+    {% if available_options|length > 0 %}
```

Comment:
Hi @brookinsconsulting 

I'm not really sure what was your point here?

All I was saying was that you should check if both `fieldSettings.options` and `field.value.identifiers` are empty before iterating over them, to make sure there are no template errors.

Also, I would not print `No selection available` if the field is empty, the choice of what should be printed out should be left to users, right?

My proposed version of template is this (I also incorporated using the delimiter throughout the template):

```twig
{% block sckenhancedselection_field %}
{% spaceless %}

{% set available_options = fieldSettings.options %}
{% set identifiers = field.value.identifiers %}

{% if fieldSettings.delimiter is not empty %}
    {% set delimiter = fieldSettings.delimiter %}
{% else %}
    {% set delimiter = ',' %}
{% endif %}

{% if available_options is not empty and identifiers is not empty %}
    {% for option in available_options %}
        {% if option.identifier in identifiers %}
            {{ option.name }}{% if not loop.last %}{{ delimiter }} {% endif %}
        {% endif %}
    {% endfor %}
{% elseif identifiers is not empty %}
    {% for identifier in identifiers %}
        {{ identifier }}{% if not loop.last %}{{ delimiter }} {% endif %}
    {% endfor %}
{% endif %}

{% endspaceless %}
{% endblock %}
```